src/Annotation/Field.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/**
+ * @author Szyon Kunowski <szymon.kunowski@gmail.com>
+ */
+
+namespace Opstalent\CrudBundle\Annotation;
+
+use Doctrine\Common\Annotations\Annotation;
+use Opstalent\CrudBundle\Annotation\Traits\ActionTrait;
+
+/**
+ * Class Field
+ * @package Opstalent\CrudBundle
+ * @Annotation
+ * @Target("PROPERTY")
+ */
+class Field
+{
+    use ActionTrait;
+
+    /**
+     * @var string[]
+     * @Required
+     * @AnnotatedDescription("The property holds all available actions for class")
+     */
+    public $actions = [];
+
+    /**
+     * Entity constructor.
+     *
+     * @param array $data
+     */
+    public function __construct(array $data)
+    {
+        if (array_key_exists("actions", $data)) {
```

Comment:
code copying. think about some `AbstractAnnotation` instead of `Trait`

Reply:
fixed